
geometry_msgs/Twist initToMarker

geometry_msgs/Twist droneToMarker

geometry_msgs/Twist initToDrone

int32 markerID